AY2223S1-CS2103-W14-1/tp
# Fix Unit Tests

Fixed syntax errors with our JUnit tests. Closes https://github.com/AY2223S1-CS2103-W14-1/tp/issues/51

To test:
1. Run `./gradlew check coverage` in terminal. 
2. You should see no syntax/checkstyle errors, however tests are still failing.

I had trouble figuring out why some tests were failing :( But maybe this can be an issue we look into in the future?
